components/phone-lottie.tsx

```
@@ -0,0 +1,377 @@
+    if (a === 0) return false;
+    const max = Math.max(r, g, b);
+    const min = Math.min(r, g, b);
+    // keep pure white and near white
+    if (max > 0.98 && min > 0.95) return false;
+    // keep pure black and near black for outlines
+    if (max < 0.05 && min < 0.05) return false;
+    return true;
+  }
+
+  function blendAngle(a: number, b: number, t: number): number {
+    let delta = ((b - a + 540) % 360) - 180; // shortest path
```

ESLint

[error] 'delta' is never reassigned. Use 'const' instead.

(prefer-const)